app/models/manageiq/providers/vmware/cloud_manager/vm.rb

```diff
@@ -22,4 +22,6 @@ def self.calculate_power_state(raw_power_state)
   def self.display_name(number = 1)
     n_('Instance (VMware vCloud)', 'Instances (VMware vCloud)', number)
   end
+
+  supports :snapshots
```

Comment:
Move this to line 3 (add empty lines) as otherwise it is really hard to spot that the VM supports this.